core-it-suite/src/test/java/org/apache/maven/it/TestSuiteOrdering.java

```diff
@@ -42,46 +44,34 @@ public class TestSuiteOrdering implements ClassOrderer
 
     final Map<Class<?>, Integer> tests = new HashMap<>();
 
-    private static void infoProperty( PrintStream info, String property )
+    private static void infoProperty( Writer info, String property ) throws IOException
     {
-        info.println( property + ": " + System.getProperty( property ) );
+        info.append( property ).append( ": " ).append( System.getProperty( property ) );
```

Comment:
Wthere is the line separator?

Reply:
revert to PrintStrem ... not need improvement